controllers/items.go

```diff
@@ -0,0 +1,55 @@
+package controllers
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"strconv"
+
+	"go-rest-api/services"
+
+	"github.com/gorilla/mux"
+)
+
+func GetItems(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	items, err := services.GetItems()
+	if err != nil {
+		errorResponse(w, "Some Error Occurred", 500)
```

Comment:
Remember that the http package already have constants for each of the status codes, you would like to change that maigc `500` number to the `http.InternalServerErrorStatus`(I think it was like that) constant.